contracts/Challenge.sol

```diff
@@ -36,18 +54,13 @@ contract Challenge {
     address payable challenger;
     uint256 blockNumberN;
   }
-  mapping(uint256 => Chal) challenges;
 
-  constructor(IMIPS imips, bytes32 globalStartState) {
-    owner = msg.sender;
-    mips = imips;
-    mem = imips.m();
-    GlobalStartState = globalStartState;
-  }
+  mapping(uint256 => Chal) challenges;
 
-  // allow getting money (and withdrawing the bounty, honor system)
-  fallback() external payable {}
+  // Allow sending money to the contract (without calldata).
   receive() external payable {}
+
+  // Allows the owner to withdraw funds from the contract.
   function withdraw() external {
     require(msg.sender == owner);
     owner.transfer(address(this).balance);
```

Comment:
Maybe use here .call? if the owner is a multi-sig will cause issues to withdraw using .transfer due to the 21k gas limit.

Reply:
Good point, I'll change that.

Reply:
Feel free to open a pull request for that actually, if you'd like (also include the other transfer below) and I'll rebase on top of it. Otherwise I'll just fix it myself, that's fine too!

Reply:
Will open a PR

Reply:
https://github.com/ethereum-optimism/cannon/pull/58